components/brave_talk/browser/regions.h

```diff
@@ -3,18 +3,17 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#ifndef BRAVE_COMPONENTS_BRAVE_TOGETHER_BROWSER_REGIONS_H_
-#define BRAVE_COMPONENTS_BRAVE_TOGETHER_BROWSER_REGIONS_H_
+#ifndef BRAVE_COMPONENTS_BRAVE_TALK_BROWSER_REGIONS_H_
+#define BRAVE_COMPONENTS_BRAVE_TALK_BROWSER_REGIONS_H_
 
 #include <string>
 #include <vector>
 
-namespace brave_together {
+namespace brave_talk {
 
 // Brave Talk is supported for all :)
-const std::vector<std::string> unsupported_regions = {
-};
+const std::vector<std::string> unsupported_regions = {};
```

Comment:
can we drop this? or at least make it via ```base::NoDestructor```?

Reply:
I deleted the file - we can always re-add if we ever need to do a restriction 😄 Ready for re-review